plugins/debug-console.js

```diff
@@ -71,34 +76,35 @@ debugTab.console.renderLine = function(errorType, args) {
   var d = new Date();
   var ta = d.toLocaleTimeString(); // print line instead maybe?
   var tb = d.toLocaleString();
-  var t = '<time title="'+tb+'" data-timestamp="'+d.getTime()+'">'+ta+'</time>';
-  var s = 'style="color:'+color+'"';
-  var l = '<tr><td>'+t+'</td><td><mark '+s+'>'+errorType+'</mark></td><td>'+text+'</td></tr>';
-  $('#debugconsole table').prepend(l);
+  var t = '<time title="' + tb + '" data-timestamp="' + d.getTime() + '">' + ta + '</time>';
```

Comment:
Could you rather use the string block for all this?

Reply:
in fact, it could also be dom element instead of string

Reply:
Well, anything will be better than this 🤷😂